snap/snapcraft.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+name: spack
+title: Spack
+icon: data/spack-logo.svg
+version: 0.19.2
+base: core22
+adopt-info: spack
+summary: A flexible package manager that supports multiple configurations.
+description: |
+  Spack is a multi-platform package manager that builds and installs multiple
+   versions and configurations of software. It works on Linux, macOS, and many
+   supercomputers. Spack is non-destructive: installing a new version of a
+   package does not break existing installations, so many configurations of the
+   same package can coexist.
+
+  Spack offers a simple "spec" syntax that allows users to specify versions and
+   configuration options. Package files are written in pure Python, and specs
+   allow package authors to write a single script for many different builds of
+   the same package. With Spack, you can build your software all the ways you
+   want to.
+license: Apache-2.0
+website: https://spack.io/
+grade: stable
+confinement: classic
+
+apps:
+  spack:
+    command: bin/spack
+    environment:
+      SPACK_USER_CONFIG_PATH: /opt/spack
+    plugs:
+      - home
+      - network
+
+parts:
+  spack:
+    plugin: dump
+    source: >-
+      https://github.com/spack/spack/releases/download/v$SNAPCRAFT_PROJECT_VERSION/spack-$SNAPCRAFT_PROJECT_VERSION.tar.gz
+    override-prime:
+      snapcraftctl prime
```

Comment:
I think this command should be `craftctl default`. It was the new standard introduced with core22:

```suggestion
      craftctl default
```

Reply:
I had used craftctl in glauth-snap but couldn't remember the updated command or find the page where that announcement was 
![image](https://user-images.githubusercontent.com/108831013/234993001-9a5782d4-c246-41e1-ad7c-08615250b647.png)
. Thanks!